src/archiefvernietigingscomponent/destruction/models.py

```diff
@@ -384,6 +384,13 @@ class ArchiveConfig(SingletonModel):
         ),
         blank=True,
     )
+    days_until_expiry = models.PositiveIntegerField(
+        _("days until expiry"),
+        default=48,
+        help_text=_(
+            "Number of days until an email is sent reminding that the list needs to be dealt with"
+        ),
```

Comment:
I think maybe the name `days_until_reminder` would be more appropriate?
Also, not sure about this, but I think 48 days is a bit long. Maybe lets put 7?

Reply:
sure, will do!